cmd/nvidia-ctk/config/config.go

```diff
@@ -71,10 +75,21 @@ func (m command) build() *cli.Command {
 			Destination: &opts.Config,
 		},
 		&cli.StringSliceFlag{
-			Name:        "set",
-			Usage:       "Set a config value using the pattern key=value. If value is empty, this is equivalent to specifying the same key in unset. This flag can be specified multiple times",
+			Name: "set",
+			Usage: "Set a config value using the pattern 'key[=value]'. " +
+				"Specifying only 'key' is equivalent to 'key=true' for boolean settings. " +
+				"This flag can be specified multiple times, but only the last value for a specific " +
+				"config option is applied. " +
+				"If the setting represents a list, the elements are colon-separated.",
 			Destination: &opts.sets,
 		},
+		&cli.StringFlag{
+			Name:        "set-list-separator",
```

Comment:
I think the `set` is redundant. When we specify `--list-separator` , it is pretty much implied that we intend to use the argument as the list separator.

Reply:
The `set` is not reduntant in this case as it refers to the `set` flag. We are specifying the list separator for the set flag.

Reply:
Ohh got it.